### PR TITLE
Feature/remove unused encoding level

### PR DESCRIPTION
**Pull request by sarwik**
* Change mapping of bib encodingLevel. Set obsolete or OCLC specific encodingLevels to marc:AbbreviatedLevel.
* Update existing data according to the new mapping
* Align this PR with https://github.com/libris/definitions/pull/119
* See LXL-2545 and LXL-2546 for more info.

### CODE REVIEW

File: whelk-core/src/main/resources/ext/marcframe.json [hunk 1]
```diff
@@ -116,18 +116,18 @@
...
-        "j": "marc:DeletedRecordLocal",
-        "k": "marc:LessThanFullLevelInputByOclcParticipantsLocal",
-        "l": "marc:FullLevelInputAddedFromABatchProcessLocal",
-        "m": "marc:LessThanFullLevelInputAddedFromABatchProcessLocal"
+        "i": "marc:AbbreviatedLevel",
+        "j": "marc:AbbreviatedLevel",
+        "k": "marc:AbbreviatedLevel",
+        "l": "marc:AbbreviatedLevel",
+        "m": "marc:AbbreviatedLevel",
+        "3": "marc:AbbreviatedLevel"
```
Comment (sarwik) [inline whelk-core/src/main/resources/ext/marcframe.json]: I want to discuss this part: To be able to make marc:AbbreviatedLevel revert to 3, I moved the declaration of token 3 to the end of list. We would need a more fullproof way of ensuring that marc:AbbreviatedLevel always reverts to 3. Or is this enough?

[the remark this answers is not in the document]
Comment (niklasl) [reply whelk-core/src/main/resources/ext/marcframe.json]: We've implemented a mechanism for reverseTokenMapOverrides which we now use here.

File: whelktool/scripts/2019/05/delete-unused-encodingLevel.groovy [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+selectBySqlWhere("""
+        collection in ('bib') AND
+        data#>>'{@graph,0,encodingLevel}' = 'marc:FullLevelInputByOclcParticipantsLocal' OR
+        data#>>'{@graph,0,encodingLevel}' = 'marc:LessThanFullLevelInputAddedFromABatchProcessLocal' OR
+        data#>>'{@graph,0,encodingLevel}' = 'marc:FullLevelInputAddedFromABatchProcessLocal' OR
+        data#>>'{@graph,0,encodingLevel}' = 'marc:LessThanFullLevelInputByOclcParticipantsLocal' OR
+        data#>>'{@graph,0,encodingLevel}' = 'marc:DeletedRecordLocal' OR
+        data#>>'{@graph,0,encodingLevel}' = 'marc:SystemIdentifiedMarcErrorInABatchloadedRecordLocal'
+    """) { data ->
```
Comment (niklasl) [inline whelktool/scripts/2019/05/delete-unused-encodingLevel.groovy]: Do we need to change "marc:CoreLevel" as well?

Comment (sarwik) [reply whelktool/scripts/2019/05/delete-unused-encodingLevel.groovy]: After checking with Tomas and Lotta we agreed to keep existing marc:CoreLevel but to disable the value in the viewer. At import of data with marc:CoreLevel the value will be set to marc:AbbreviatedLevel instead. All existing instances with marc:CoreLevel will revert to token 3 (marc:AbbreviatedLevel).

Comment (niklasl) [reply whelktool/scripts/2019/05/delete-unused-encodingLevel.groovy]: Sounds good!

**Pull request merged**